sys/net/routing/rpl/rpl.c

```diff
@@ -279,32 +273,16 @@ void *rpl_process(void *arg)
 void _rpl_update_routing_table(void)
 {
```

Comment:
@cgundogan the FIB works completely without threads. The exact invalidation point in time of a route is fixed and known, but not monitored by the FIB. A route lifetime and its invalidation is only performed on demand when the specific FIB entry is touched.
So if the lifetime of a route exceeds, probably the parent invalidation will not be triggered.